app/models/emaillink.rb

```diff
@@ -0,0 +1,3 @@
+class Emaillink < ActiveRecord::Base
```

Comment:
I kept typing this as `EmailLink` instead of `Emaillink`. It's completely up to you, but my inclination is to change the name to `EmailLink`.